mobile-widgets/qml/TemplateButton.qml

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
Can we do a single commit that does indeed do the git move and replaces all the uses?
I think that would be preferrable.

Reply:
Done.